lighthouse-core/test/lib/sentry-test.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+jest.mock('raven');
```

Comment:
what's the unmocking behavior for this? The docs don't really make it clear. Does it rely on the tests being run in different processes so that other requires won't get a mock version?

Reply:
even in the same process `jest` has a require sandbox that require is not real require and does auto-mocking of all kinds of things so there should be no need to unmock elsewhere (though even if it were mocked everywhere, I wouldn't be disappointed in the extra layer of assurance its not going to sentry :) )

Reply:
ah yeah, less worried about sentry leaking in other tests than just how they work in general :)